### ISSUE TITLE
Orebfuscator not working

### ISSUE THREAD

**Issue by WinSide** [CyberdyneCC/Thermos#466]
This server is running Thermos | https://github.com/CyberdyneCC/Thermos | 1.7.10-R0.1-SNAPSHOT | 1.7.10-1614.57 (MC: 1.7.10)
Errors on pastebin
http://pastebin.com/LtgMdT1r

**Comment by spannerman79** [comment]
This is a known fact - you cannot run any sort of anti xray or "Orebfuscator" on any cauldron or fork of cauldron server.

First and foremost Cauldron and its forks is primary a forge based instance with *limited* bukkit capabilities.  Even the forge component has limitations due to the merge and remapping required to get both forge and bukkit to play in the same sandbox so to speak.

Back even when bloodmc was active with cauldron someone asked about this same issue and the responce was very colourful on the (not defunct) cauldron forums.

In short - don't bother as it doesn't work with modded ores - hence the errors.

**Comment by WinSide** [comment]
It's so bad.. Please close

**Comment by spannerman79** [comment]
You can close your own issue yourself.
